experimental/StandardFiniteFields/test/runtests.jl

```diff
@@ -0,0 +1,44 @@
+using Test
+using Oscar
+GAP.Packages.load("StandardFF")
+
+function MyPoly(p,n)
+    return defining_polynomial(standard_finite_field(p,n))
+end
+
+function GAPPoly(p,n)
+    poly = GAP.evalstr( "StandardFiniteField(" * string(p) * ", " * string(n) * ")!.DefiningPolynomial!.CoefficientsOfUnivariatePolynomial")
```

Comment:
```suggestion
    poly = GAP.evalstr("StandardFiniteField($(p), $(n))!.DefiningPolynomial!.CoefficientsOfUnivariatePolynomial")
```
easier to read

Reply:
Better yet, don't use `evalstr` (it is almost never necessary and should be avoided). Also don't access internal fields of objects if possible, use the proper accessors instead. So:
```suggestion
    poly = GAP.Globals.StandardFiniteField(p, n)
    poly = GAP.Globals.DefiningPolynomial(poly)
    poly = GAP.Globals.CoefficientsOfUnivariatePolynomial(poly)
```

Reply:
Thanks Lars, Max, this cleans it up quite nicely